bin/docker-install-tutorials.sh

```diff
@@ -1,23 +1,67 @@
-# install everything for all tutorials of a topic
+#!/bin/bash
 
-set -e
-galaxy_instance="http://localhost:8080"
+# set API key
+API_KEY=${GALAXY_DEFAULT_ADMIN_KEY:-fakekey}
 
-# launch the instance
-echo " - Starting Galaxy.. \n"
+# Enable Test Tool Shed
+echo ".. Setting up conda"
+export GALAXY_CONFIG_TOOL_SHEDS_CONFIG_FILE=$GALAXY_HOME/tool_sheds_conf.xml
 
-export GALAXY_CONFIG_TOOL_PATH=/galaxy-central/tools/
-startup_lite
+. /tool_deps/_conda/etc/profile.d/conda.sh
+conda activate base
 
-# wait until galaxy has started
-galaxy-wait -g $galaxy_instance
+if pgrep "supervisord" > /dev/null
+then
+    echo ".. System is up and running. Starting with the installation."
+    export PORT=80
+else
+    # start Database
+    echo ".. Starting database"
+    export PORT=8080
+    service postgresql start
+    install_log='galaxy_install.log'
 
-# run tutorial install as user galaxy
-su - $GALAXY_USER
+    # wait for database to finish starting up
+    STATUS=$(psql 2>&1)
```

Comment:
shouldn't `psql` hang if it's alive/working? Maybe `psql -l` would be better since you know that command terminates?

Reply:
Huh! That's unexpected. `psql` just opens an interactive postgres session, so it intentionally hangs forever waiting for user input. If it works there (and here) then no reason to change it. Just was surprising.